lib/dynflow/action/v2/with_sub_plans.rb

```diff
@@ -173,6 +175,7 @@ def concurrency_limit_capacity
     def cancel!(force = false)
       # Count the not-yet-planned tasks as cancelled
       output[:cancelled_count] = total_count - output[:planned_count]
+      output[:cancelled_timestamp] = DateTime.now.new_offset(0).to_s # time in UTC for comparison with UTC times in the database
```

Comment:
I'd rather be explicit with the format here to save ourselves from any suprises
```suggestion
      output[:cancelled_timestamp] = DateTime.now.new_offset(0).iso8601 # time in UTC for comparison with UTC times in the database
```

Reply:
Also this reads easier (or at least to me)
```suggestion
      output[:cancelled_timestamp] = Time.now.utc.iso8601 # time in UTC for comparison with UTC times in the database
```